content/source/docs/enterprise/admin/agents-on-tfe.html.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: "enterprise"
+page_title: "Terraform Cloud Agents on TFE - Infrastructure Administration - Terraform Enterprise"
+---
+
+# Terraform Cloud Agents on TFE
+
+Terraform Cloud Agents allow Terraform Enterprise to communicate with isolated,
+private, or on-premises infrastructure. By deploying lightweight agents within a
+specific network segment, you can establish a simple connection between your
+environment and Terraform Enterprise which allows for provisioning operations and
+management.
+
+Terraform Cloud Agents on TFE behave very similarly to [Terraform Cloud Agents on TFC](https://www.terraform.io/docs/cloud/agents/index.html), with a few exceptions listed below. This page contains the steps to get started and operational requirements for Terraform Enterprise. 
```

Comment:
One more tiny nit - does this contain the steps to get started? I think it's largely just differences. Maybe we omit "steps to get started" so it's clearer?

Reply:
Sorry, i think i got a few things backwards. The intent of this is to say, "the page at Terraform Cloud Agents on TFC is the primary source of documentation, and _this current_ page ('Terraform Cloud Agents on TFE') contains TFE-specific information". Right now I don't feel like this is being well-conveyed; i'm gonna rework it. sorry for confusing things!

Reply:
No worries - that makes sense! I just took a look and I like the new version!